src/panels/CharmsPanel/CharmsPanel.tsx

```diff
@@ -38,41 +32,32 @@ export default function CharmsPanel({
 
   return (
     <Panel
+      drawer={
+        <Button
+          disabled={!selectedCharm}
+          onClick={() => onCharmURLChange(selectedCharm)}
+        >
+          Next
+        </Button>
+      }
       width="narrow"
-      panelClassName={ClassName.PANEL}
       data-testid={TestId.PANEL}
       title={Label.PANEL_TITLE}
       onRemovePanelQueryParams={onRemovePanelQueryParams}
       loading={isLoading}
     >
-      <div className="p-panel__content p-panel_content--padded">
```

Comment:
Again, mostly just indentation changes except for moving the buttons to the drawer.